web/client/components/misc/enhancers/handleLongTextEnhancer.jsx

```diff
@@ -0,0 +1,35 @@
+/*
+* Copyright 2023, GeoSolutions Sas.
+* All rights reserved.
+*
+* This source code is licensed under the BSD-style license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+import React from "react";
+import OverlayTrigger from "../OverlayTrigger";
+import { Tooltip } from "react-bootstrap";
+
+export  const handleLongTextEnhancer = (RenderFormatter) => (props) => {
+    const { value } = props;
+    const cellRef = React.useRef(null);
+    const contentRef = React.useRef(null);
+    const [isContentOverflowing, setIsContentOverflowing] = React.useState(false);
+
+    const handleMouseEnter = () => {
+        const cellWidth = cellRef.current.offsetWidth;
+        const contentWidth = contentRef.current.offsetWidth;
+        setIsContentOverflowing(contentWidth > cellWidth);
+    };
+
+    return (<OverlayTrigger
+        placement="top"
+        overlay={isContentOverflowing ? <Tooltip id="tooltip">{RenderFormatter ? <RenderFormatter {...props} /> : value}</Tooltip> : <></>}
```

Comment:
is the ternary needed? i mean don't we always have the Wrapped component when using this enhancer?

if it is not needed we can remove it here and in the below